notebooks/214-grammar-correction/214-grammar-correction.ipynb

```diff
@@ -0,0 +1,500 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    print(f'predicted label: {"contains_errors" if result["label"] == "LABEL_1" else "no errors"}, score: {result["score"] :.2}')
</pre><p>Could we print the score in a separate line?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/839/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
moved